docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,104 @@
+# Convert ONNX\* RNN-T Model to the Intermediate Representation (IR) {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
```

Comment:
You need to update `docs/doxygen/ie_docs.xml` otherwise your new document will not appear in the documentation.

Reply:
done